internal/msg/msg.go

```diff
@@ -63,6 +64,23 @@ https://bit.ly/saucectl-signup
 Already have an account? Get your username and access key here:
 https://app.saucelabs.com/user-settings`
 
+// SauceIgnoreNotExist is a recommendation to create a .sauceignore file in the case that it is missing.
+const SauceIgnoreNotExist = `The .sauceignore file does not exist. We *highly* recommend creating one so that saucectl does not
+create archives with unnecessary files. You are very likely to experience longer startup times.
+
+For more information, visit https://docs.saucelabs.com/testrunner-toolkit/configuration/bundling/index.html#exclude-files-from-the-bundle
+
+or peruse some of our example repositories:
+  - https://github.com/saucelabs/saucectl-cypress-example
```

Comment:
Mostly just thinking out loud, is there a benefit to maybe link directly to the example `.sauceignore` file?

Reply:
The thought _did_ indeed cross my mind. But directly linking files is also more liable to change (kinda like the docs link that is bound to change).

Unless you don't link to  `https://github.com/saucelabs/saucectl-cypress-example/blob/master/.sauceignore` but instead use a permalink like `https://github.com/saucelabs/saucectl-cypress-example/blob/c0b6b30bb7309ccf093f2c7c79037ee155568af9/.sauceignore`.

That one is even less sexy during a console printout though 🤔

Reply:
Yah, that hash is fugly, good call.

One of my favourite clis is the elm-lang compiler. It also uses links extensively in its output which is great. The compiler output links to markdown files per command: https://github.com/elm/compiler/tree/master/hints

Bringing that up only because this PR reminded me of that. Just something to chew on. 😄